frontend/src/components/CostTableMenu.vue

```diff
@@ -105,14 +100,49 @@ const EngineIconPath = {
   POSTGRES: new URL("../assets/icon/db-postgres.png", import.meta.url).href,
 };
 
+const dropdownOptionList = computed(() => {
+  const dropDownList: any = [];
+  const parentKey = new Set<string>();
+
+  availableRegionList.forEach((region) => {
+    const parentRegion = region.substring(0, region.indexOf("(") - 1);
+    const childRegion = region.substring(
+      region.indexOf("(") + 1,
+      region.indexOf(")")
+    );
```

Comment:
We don't know what a `region` looks like. I feel that it's something like `parent(child)` but I think we need to address more comments here.
Then we may also consider matching with regular expression here or just using `indexOf` like now.

Reply:
You are right, I should adress this at the backend and defined a better structure to represent this.
Will improve this via another PR.